examples/dragBeamReconfiguration/dragBeamReconfiguration.m

```diff
@@ -325,3 +325,87 @@
+elementP0 = (2 * rand(numElements, 1) - 1) * 0.001;
+initialConds.P0 = repelem(elementP0, dofsWakeVariablesPerElement);
+% md
+% md### analysisSettings
+% md
+% md The $\alpha$-HHT algorithm is set with the following tolerances, time step, and final time:
+analysisSettings = struct();
+analysisSettings.finalTime = 0.7;
+% analysisSettings.finalTime =   5.0; % to reproduce annimation
+analysisSettings.deltaT = 0.01;
+analysisSettings.methodName = 'alphaHHT';
+analysisSettings.stopTolIts = 50;
+analysisSettings.stopTolDeltau = 0;
+analysisSettings.stopTolForces = 1e-5;
+% md The constasnt veliocity field corresponding to `NR - 3` step is set as well as density and viscvotiy of the fluid:
+analysisSettings.fluidProps = {rhoF; nuF; 'windVelCircDynamic'};
+% md The following parameters are set to configure the dynamic analysis considering VIV. The `analysisSettings.crossFlowVIVBool` parameter enables the consideration of cross-flow VIV in the analysis. The `analysisSettings.inLineVIVBool` parameter determines whether in-line VIV is considered, and it is set to `true` in this case. Lastly, the `analysisSettings.addedMassBool` parameter accounts for the added mass effect on fluid forces.
+analysisSettings.crossFlowVIVBool = true;
+analysisSettings.inLineVIVBool = true;
+analysisSettings.addedMassBool = true;
+% md### otherParams
+% md The name of the problem is set and VTK output is exported to generate an animation:
+otherParams = struct();
+otherParams.problemName = 'vivBeamReconfiguration';
+otherParams.plots_format = 'vtk';
+% md
+% md### Numeric solution
+% md
+% Initialize and solve the dynamic problem
+[modelCurrSol, modelProperties, BCsData] = initONSAS(materials, elements, boundaryConds, initialConds, mesh, analysisSettings, otherParams);
+% Perform the dynamic analysis
+[matUsDynamic, loadFactorsMat, modelSolutions] = solveONSAS(modelCurrSol, modelProperties, BCsData);
+% md
+% md Evolution of displacements of node A
+% md
+finalNodeIndex  = numElements + 1;
+zDisplacement   = matUsDynamic(5:6:end, :);
+timeVector      = 0:analysisSettings.deltaT:analysisSettings.finalTime;
+% Generate figure
+figZDisplacement = figure;
+plot(timeVector, zDisplacement(finalNodeIndex, :), 'b-', 'LineWidth', 2);
+xlabel('Time [s]');
+ylabel('Displacement in z [m]');
+title('Displacement in z at A');
+grid on;
+set(gca, 'linewidth', axislw, 'fontsize', curveFontSize);
+set(get(gca, 'xlabel'), 'FontSize', axisFontSize);
+set(get(gca, 'ylabel'), 'FontSize', axisFontSize);
+% Save figure for automatic deployment
+namefigZ = strcat(folderPathFigs, 'zDisplacementVIV.png');
+if length(getenv('TESTS_RUN')) > 0 && strcmp(getenv('TESTS_RUN'), 'yes')
+  fprintf('\ngenerating output png for docs.\n');
+  print(figZDisplacement, namefigZ, '-dpng');
+else
+  fprintf('\n === NOT in docs workflow. ===\n');
+end
+% md
+% md```@raw html
+% md<img src="../../assets/generated/zDisplacementVIV.png" alt="Displacemtns in z direction of node A" width="500"/>
```

Comment:
I am not seeing the plots and the gif in the PR's deployment. However, since running vtk output takes time I suggest leaving the vtk output and also the plot for the NON-server case using the tests_run env. and upload the plots generated locally to the docs assets folders